Lib/extractor/formats/opentype.py

```diff
@@ -48,6 +67,20 @@ def extractFontFromOpenType(
         extractOpenTypeInfo(source, destination)
     if doGlyphs:
         extractOpenTypeGlyphs(source, destination)
+        if isVarFont:
+            '''
+            Add Support for extracting Variable Instances as RLayer objects
```

Comment:
Better separate UFOs or layers? Not sure.

Reply:
Oh good point, I had imagined a layer-based import just because I thought containing everything in one file was more "UFO-esque" but multiple files could be a better solution.

Reply:
May be good to have the option to do either

Reply:
Extractor works by providing a dest font object and a source binary file. 

```python
ufo = defcon.Font()
extractor.extractUFO("/path/to/MyFont.ttf", ufo)
```

Returning a list of ufo's is a big change. Adding arguments is also a big change.

Im thinking that extracting variable fonts should be implicit, and maybe directly into a designspace object: extract axis, ranges, the relationships between sources, store font objects as sources in the designspace document object:

```python
extractVariableFont("path/to/varfont.ttf", designspaceDocumentObject)
# raise error when no fvar table is found
```

Extractor is not build to round trip from binary to ufo and back, so parts can lost during extracting. But is would be cool to get as much data as possible back into scriptable objects.